.github/workflows/ci.yml

```diff
@@ -781,24 +781,25 @@ jobs:
           change=$([ $current_size -gt $base_size ] && echo "increased❗" || ([ $current_size = $base_size ] && echo "not changed" || echo "decreased✅"))
           echo "change=${change}" >> $GITHUB_OUTPUT
           echo "diff=$(($current_size - $base_size))" >> $GITHUB_OUTPUT
-      - name: Find Comment
-        uses: peter-evans/find-comment@v2
-        id: fc
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          comment-author: 'github-actions[bot]'
-          body-includes: '## ${{ matrix.app }} bundle size is'
-      - name: Create or update comment
-        uses: peter-evans/create-or-update-comment@v2
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Update PR Description
+        uses: nefrob/pr-description@v1.1.2
```

Comment:
Does `nefrob/pr-description@v1.1.2` apply sanitization?

Reply:
Unfortunately,  
nefrob/pr-description@v1.1.2 does not handle sanitization:
https://github.com/nefrob/pr-description/blob/master/index.js

Reply:
Looking at the code it appears to only use `@actions/core` libraries and does not do parsing itself (except for the `replace`).
This falls inline with:

> Prefer action over inline script. Action’s parameters are stringified, which prevents code injection by treating the input as text data, making it harmless even if it contains malicious code.

https://openssf.org/blog/2024/08/12/mitigating-attack-vectors-in-github-workflows/

i.e. It doesn't appear to execute the untrusted (pr desciption) strings (unlike doing something like `git checkout branch_name` which actively passes the branch name into an execution statement).

However I am no security expert, it would be prudent to have someone in a security team have a review over whether this is acceptable.